<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWinClass</class>
 <widget class="QMainWindow" name="MainWinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Senior Timer-1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res.qrc">
    <normaloff>:/pictures/qsenior-timer.ico</normaloff>:/pictures/qsenior-timer.ico</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="central_vertical_layout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="act_view_hidden"/>
    <addaction name="act_graph_analyser"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="act_manual"/>
    <addaction name="act_send_feedback"/>
    <addaction name="act_check_update"/>
    <addaction name="act_about"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="act_add_timer"/>
    <addaction name="act_settings"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>存档</string>
    </property>
    <addaction name="act_save_local"/>
    <addaction name="act_cloud_store"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="act_graph_analyser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>可视化分析</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_check_update">
   <property name="text">
    <string>检查更新</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_manual">
   <property name="text">
    <string>使用方法</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_add_timer">
   <property name="text">
    <string>添加计时器</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_settings">
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>关于 Senior Timer</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_send_feedback">
   <property name="text">
    <string>发送反馈</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_view_hidden">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>查看隐藏项</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_save_local">
   <property name="text">
    <string>保存存档(本地)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="act_cloud_store">
   <property name="text">
    <string>云存档</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
